resources/assets/uad_lists.json

```
@@ -12332,7 +12332,7 @@
   {
     "id": "com.lge.floatingbar",
     "list": "Oem",
-    "description": "LG Floating bar\nLets you put shortcuts to apps or features, as well as quick access to contacts and music player controls on a \"floating bar\" on the Home screen.\nhttps://www.neowin.net/news/lg-v30-closer-look-floating-bar/\n",
+    "description": "LG Floating bar\nLet's you put shortcuts to apps or features, as well as quick access to contacts and music player controls on a \"floating bar\" on the Home screen.\nhttps://www.neowin.net/news/lg-v30-closer-look-floating-bar/\n",
```

```suggestion
    "description": "LG Floating bar\nLets you put shortcuts to apps or features, as well as quick access to contacts and music player controls on a \"floating bar\" on the Home screen.\nhttps://www.neowin.net/news/lg-v30-closer-look-floating-bar/\n",
```
Incorrect. "Lets" is correct: https://www.grammarly.com/blog/lets-vs-lets/#:~:text=The%20difference%20between%20let's%20and%20lets&text=Let's%20is%20a%20contraction%20of%20%E2%80%9Clet%20us.%E2%80%9D%20You%20use,to%20allow%20or%20give%20permission.